voluncheer/.ebextensions/01_django.config

```diff
@@ -17,6 +17,9 @@ container_commands:
     command: "echo '*/10 * * * * root source /var/app/venv/*/bin/activate && python3 manage.py archive_opportunities' > /etc/cron.d/archive_opportunities"
   07_set_archive_permissions:
     command: "chmod 644 /etc/cron.d/archive_opportunities"
+  08_wsgipass:
+    command: 'echo "WSGIPassAuthorization On" >> ../wsgi.conf'
```

Comment:
Should we just put this file in the code instead of creating it every time we deploy?

Reply:
Doesn't seem necessary anymore.